File: python/odev1.py
MIN_DIST = 0.1
MAX_DIST = 30.0

try:
    mesafe = float(input("Mesafe girin: "))
    
    if mesafe < MIN_DIST:
        print("PARAZIT!")
    elif mesafe > MAX_DIST:
        print("COK UZAK")
    else:
        print(f"OK: {mesafe:.2f} m")
except ValueError:
    print("HATA: Gecersiz girdi!")

File: Cpp/odev1.cpp
#include <iostream>

void sensorVerisi(float mesafe) {
    if (mesafe < 0.1f) {
        std::cout << "Hatali veri: " ;
    }
    else if (mesafe > 30.0f) {
        std::cout << "Cok uzak, yoksay. " ;
    }else {
        std::cout << "Nesne algilandi. Mesafe: " << mesafe << std::endl;
    }
}

int main() {
    float veri;

    std::cout << "Metre cinsinden bir mesafe degeri girin: ";
    std::cin >> veri;
    sensorVerisi(veri);
  
    return 0;
}
